packages/communication-react/CHANGELOG.md

```diff
@@ -4,80 +4,23 @@ This log was last generated on Mon, 14 Feb 2022 23:50:05 GMT and should not be m
 
 <!-- Start content -->
 
-## [1.1.0](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.1.0)
+## [1.1.0](https://github.com/azure/communication-ui-library/tree/1.1.0)
 
-Mon, 14 Feb 2022 23:50:05 GMT 
-[Compare changes](https://github.com/azure/communication-ui-library/compare/@azure/communication-react_v1.0.1...@azure/communication-react_v1.1.0)
+Mon, 14 Feb 2022 23:50:05 GMT
 
-### Minor changes
+This minor release contains general bug fixes and performance improvements.
 
-- `@internal/chat-component-bindings`
-  - Add options param to sendMessage for sending metadata ([PR #1374](https://github.com/azure/communication-ui-library/pull/1374) by anjulgarg@live.com)
-- `@internal/react-components`
-  - created button component for local video feed camera switcher ([PR #1367](https://github.com/azure/communication-ui-library/pull/1367) by 94866715+dmceachernmsft@users.noreply.github.com)
-  - Add metadata attribute to ChatMessage type ([PR #1374](https://github.com/azure/communication-ui-library/pull/1374) by anjulgarg@live.com)
-  - Identifiers added for HorizontalGallery left and right navigation buttons. ([PR #1347](https://github.com/azure/communication-ui-library/pull/1347) by miguelgamis@microsoft.com)
-- `@internal/react-composites`
-  - Exposed Chat and Call latestErrors in MeetingComposite ([PR #1456](https://github.com/azure/communication-ui-library/pull/1456) by edwardlee@microsoft.com)
-  - Added new constructor that accepts stateful client arguments in MeetingComposite ([PR #1457](https://github.com/azure/communication-ui-library/pull/1457) by edwardlee@microsoft.com)
-  - Created component for ParticipantList and heading ([PR #1440](https://github.com/azure/communication-ui-library/pull/1440) by miguelgamis@microsoft.com)
-  - Updated calling composite icons to include new icon for local video feed camera switcher ([PR #1367](https://github.com/azure/communication-ui-library/pull/1367) by 94866715+dmceachernmsft@users.noreply.github.com)
+## Changes since 1.0.1-beta.2
+[Compare changes](https://github.com/Azure/communication-ui-library/compare/1.0.0...1.1.0)
 
-### Patches
 
-- `@internal/chat-component-bindings`
-  - Upgrade @azure/communication-signaling to 1.0.0.beta.12 ([PR #1352](https://github.com/azure/communication-ui-library/pull/1352) by anjulgarg@live.com)
-- `@internal/chat-stateful-client`
+### Changes
   - Upgrade @azure/communication-signaling to 1.0.0.beta.12 ([PR #1352](https://github.com/azure/communication-ui-library/pull/1352) by anjulgarg@live.com)
```

Comment:
Is it OK to have beta package in our stable releases?

Reply:
Good question.
Ideally not. But we have been using the beta packages for this dep even in our past stable releases unfortunately.

Reply:
There is a bug tracking this. They don't have a stable package *at all*: https://www.npmjs.com/package/@azure/communication-signaling

And I think there are some types we can't get from @azure/communication-chat instead of this one.